### PR TITLE
NOTIF-104 Save actions in the same order they are specified

### PR DESCRIPTION
**Pull request by josejulio**

### CODE REVIEW

File: src/pages/Notifications/Form/useSaveBehaviorGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                    operation: SaveBehaviorGroupResult.CREATE,
+                    status: true
+                };
+            } else {
+                return {
+                    operation: SaveBehaviorGroupResult.UPDATE,
+                    status: true
+                };
+            }
+        } else if (behaviorGroupId === undefined) {
```
Comment (aferd) [inline src/pages/Notifications/Form/useSaveBehaviorGroup.tsx:156]: Why is this in here back to back? See lines below (@ 169)

Comment (aferd) [reply src/pages/Notifications/Form/useSaveBehaviorGroup.tsx:156]: nvm, I see the return status is different.

Comment (josejulio) [reply src/pages/Notifications/Form/useSaveBehaviorGroup.tsx:156]: It's a bit difficult to see with the `if` nesting, but line 169 is outside of the `if actionsNeedsUpdate(original, updated)` at line 105.

So line 169 deals with the case when we didn't need to update, but are returning an OK response anyway.

**Review by aferd — APPROVED** [review]
lgtm

**Pull request merged**